src/NzbDrone.Core/Indexers/IndexerDefaults.cs

```diff
@@ -3,5 +3,6 @@
     public static class IndexerDefaults
     {
         public const int MINIMUM_SEEDERS = 1;
+        public const int PRIORITY = 1000; //default > 0 so it allows users to specify higher or lower prio indexers without having to go trough all of them
```

Comment:
Why does it need to be 1000 for users to set higher/lower priority? Zero as neutral makes more sense to me than any other number and a negative number to lower the priority would work fine and is more obvious.

Reply:
IMO I don't expect many users to try negative numbers since it's counter intuitive. I took 1000 because it allows users to set other indexers to 900 800 etc. and leave a gap for them to fill in (801-899) if I want to add another indexer in the middle later on (for example when they get into a new private tracker). Tbh we could also pick 100. Just my opinion, if you disagree I'll change it to 0.

Reply:
Negative to lower priority seems pretty intuitive to me and also matches preferred words in terms of negative is lower than the default. I asked @Taloth what he thinks.